tests/resources/proton_run000201_za20deg_azm000deg_North_alpha_6.0.0_test_file/config.yml

```diff
@@ -0,0 +1,26 @@
+---
+CTA_SIMPIPE:
+  APPLICATIONS:
+  - APPLICATION: simtools-simulate-prod
+    CONFIGURATION:
+      SIMULATION_SOFTWARE: corsika_sim_telarray
+      LABEL: test_file
+      MODEL_VERSION: 6.0.0
+      SITE: North
+      ARRAY_LAYOUT_NAME: alpha
+      PRIMARY: proton
+      RUN_NUMBER: 201
+      AZIMUTH_ANGLE: 0
+      ZENITH_ANGLE: 20
+      NSHOW: 50
+      VIEW_CONE: 0 deg 10 deg
+      ENERGY_RANGE: 30 GeV 600000 GeV
```

Comment:
How did you choose this energy range? Starting from 30 GeV I could understand to increase the chance for triggering (but it would be a mistake for calculating trigger rate). However, why go up to 600 TeV in such a test file?

Reply:
30 GeV are need to get some triggered events. I didn't change the upper end from the template I've used from the integration tests. Is this an issue?